TombEngine/Sound/sound.h

```diff
@@ -153,6 +153,14 @@ struct SoundSourceInfo
 	}
 };
 
+struct BassDevice {
```

Comment:
Use formatting according to coding conventions, please (no `{` on the same line).